rest-api-spec/src/main/resources/rest-api-spec/api/indices.exists_index_template.json

```diff
@@ -1,7 +1,7 @@
 {
   "indices.exists_index_template":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-templates.html",
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/index-templates.html",
```

Comment:
There's no doc page for `HEAD _index_template/{name}`, only for [`HEAD _template/{name}`](https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-template-exists-v1.html). I've considered linking to the `GET _index_template/{name}`, but instead to the general page on the topic, which is better than the old redirect page.